force-app/main/default/classes/StringGenerator.cls

```diff
@@ -0,0 +1,30 @@
+public with sharing class StringGenerator {
+
+    private List<String> my_list = new List<String>();
```

Comment:
we follow a standard for variable naming. Use camelCase, in this case it will be myList Here is the refernce: https://ortooapps.sharepoint.com/sites/OrtooDevelopmentTeam/_layouts/OneNote.aspx?id=%2Fsites%2FOrtooDevelopmentTeam%2FShared%20Documents%2FImprovements%20to%20Development%2FOrtoo%20Development%20Team%20Notebook&wd=target%28Process%2C%20standards%2C%20templates.one%7C9A1DCADC-F6CA-4F2E-A76B-9F2462414C25%2FCode%20Conventions%20%26%20Style%7C1FE4125F-C8FA-4C9C-A58E-7134FD78DD78%2F%29
onenote:https://ortooapps.sharepoint.com/sites/OrtooDevelopmentTeam/Shared%20Documents/Improvements%20to%20Development/Ortoo%20Development%20Team%20Notebook/Process,%20standards,%20templates.one#Code%20Conventions%20%20Style&section-id={9A1DCADC-F6CA-4F2E-A76B-9F2462414C25}&page-id={1FE4125F-C8FA-4C9C-A58E-7134FD78DD78}&end